shard.override.yml

```diff
@@ -0,0 +1,11 @@
+# This file is used to override the shards built by
+# generated Lucky apps. Comment these out if you want to
+# use the versions specified by these apps.
+
+# NOTE: https://github.com/crystal-lang/shards/issues/432
+# Until this is resolved, the default needs to be whatever the
+# default Lucky is.
+dependencies:
```

Comment:
I think it would be less error-prone to use this suggestion https://github.com/crystal-lang/shards/issues/432#issuecomment-683150150

I can see us running into confusion when we change shards.yml but forget to change the override.yml